sa/database.go

```diff
@@ -12,29 +12,48 @@ import (
 	"strings"
 
 	// Provide access to the MySQL driver
```

Comment:
We can now remove this comment. We used to need it to explain why we were ignoring the import.